posts/deploying-an-openshift-application-in-aws-govcloud.md

```diff
@@ -0,0 +1,120 @@
+---
+authors:
+- Dennis Sharpe
+- David Elizondo
+tags:
+- AWS
+- GovCloud
+- OpenShift
+- FIPS
+- FedRAMP
+date: 2021-12-15T12:21:50.000Z
+title: "Deploying an OpenShift Application in AWS GovCloud"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2021/12/deploying-an-openshift-application-in-aws-govcloud.png
+---
+#Context
+Our project entailed moving an existing web application from a commercial AWS account to AWS GovCloud. In addition to moving to GovCloud, we had to adhere to FedRAMP High security guidelines. The original application was never designed to run in GovCloud. We will offer some tips on how to make a move like this as smooth as possible.
+
+The original application was built using the following technologies:
+- JHipster
+- ReactJS
+- Java
+- Spring Boot
+- PostgreSQL
+- Kafka
+- Redis
+- OKD (Open Source RedHat OpenShift)
+
+The target environment required:
+- FedRAMP High Compliance
+- RedHat OpenShift
+- AWS GovCloud
+
+#Additional Requirements
+For security purposes, we limited the connections from GovCloud out to the internet. We also created our own repository in GovCloud. Lastly, we installed Nexus in GovCloud. However, it only connects to the development environment Nexus to download *only* pre-approved libraries. Of course, we made a few exceptions when necessary, such as downloading libraries from a trusted source like RedHat.
```

Comment:
>  However, it only connects to the development environment Nexus to download only pre-approved libraries. Of course, we made a few exceptions when necessary, such as downloading libraries from a trusted source like RedHat.

reword to: 
However, *only* pre-approved libraries are downloaded from the development environment Nexus.

Or something like that. In this sentence  is *it* a pipeline building an executable? Also using *only* twice reads a little awkward